ddemangle.d

```diff
@@ -25,16 +25,18 @@ Demangles all occurrences of mangled D symbols in the input and writes to
 standard output.
 If <inputfile> is omitted, standard input is read.
 Options:
-    --help, -h    Show this help
+    --help, -h                  Show this help
+    --underscore_missing, -u    handling missing underscore (eg with lldb on OSX)
```

Comment:
Would need to update https://dlang.org/rdmd.html and the manual too.

Reply:
https://dlang.org/rdmd.html  makes no mention of ddemangle do you mean another link?
man ddemangle also doesn't show anything (for me); can you point me to source code perhaps?

Reply:
Ah sorry - somehow I was thinking that we had man pages and docs for ddemangle. Nevermind then.

For reference, the man pages are stored here - https://github.com/dlang/dmd/tree/master/docs 
-> https://issues.dlang.org/show_bug.cgi?id=18400